src/js/letters/reducers/index.js

```diff
@@ -104,6 +108,13 @@ function letters(state = initialState, action) {
       return _.set(['letterDownloadStatus', action.data], 'failure', state);
     case UPDATE_ADDRESS:
       return _.set('address', action.address, state);
+    case SAVE_ADDRESS_PENDING:
+      return _.set('savePending', true, state);
```

Comment:
This currently doesn't do anything, but I'm tracking that [here](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/4859).